File: corte1 /Ejercicios(1)/Ejemplo1/C++/Person/Person.cpp
#include <iostream>
#include <string>
using namespace std;

class Person {
public:
    string name; //Atributo
    int age; // atributo

    void sayHello() {
        cout << "Hola! Soy " << name << " Y tengo" << age << "años.\n";
            
    }
};

int main() {
    Person p1;
    p1.name = "Alex";
    p1.age = 18 ;
    p1.sayHello();
    return 0;
}

File: corte1 /Ejercicios1/Practicas1/Ejemplo1/Python/person/person.py
class person:
    def __init__(self):
        self.name = " "
        self.age = 0
        
    def say_hello (self):
        print(f"Hola, soy {self.name} y tengo {self.age} años!")
def main():
    p1 = person()
    p1.name = "Alex"
    p1.age = 18
    p1.say_hello()
    
if __name__ == "__main__":
    main()        